.travis.yml

```diff
@@ -29,7 +29,7 @@ before_install:
   - sudo mysql -e "ALTER DATABASE exercism_reboot_test CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;" -u root
   - sudo mysql -e "GRANT ALL PRIVILEGES ON exercism_reboot_test.* TO 'exercism_reboot'@'localhost'" -u root
 install:
-  - "bundle install"
+  - "bundle install --jobs=3 --retry=3 --deployment"
```

Comment:
This command is what Travis CI uses in order to install dependencies. It uses parallel jobs to make it faster, and also adds the `--deployment` flag in order to write the gems into `vendor/bundle`.

As we see below in this file, we cache `vendor/bundle` but we aren't able to cache anything because we left out the `--deployment` flag.